content/SCALE/SCALECLIReference/_index.md

```diff
@@ -136,6 +136,44 @@ These option=value pairs are enclosed in curly brackets, but can also include sq
 
 Each namespace article includes command syntax examples for each namespace.
 
+### Using the Interactive Arguments Editor
+
+Enter the `--` option flag following any CLI command to open the **interactive arguments editor** text-based user interface (TUI).
```

Comment:
We decided not to use "option" so recommend either removing it or replace with property